RecoEgamma/EgammaPhotonProducers/python/reducedEgamma_cfi.py

```diff
@@ -52,6 +54,14 @@
         preshowerEcalHits = cms.InputTag(""),
 )
 
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(
```

Comment:
```suggestion
pp_on_AA.toModify(
```

I think that this should use `pp_on_AA`
these two are supposed to be additional modifiers for year/period-specific  items

Reply:
I just copied this code from https://github.com/cms-sw/cmssw/pull/30255/files#diff-f273dbbb39fe5fe2475dabb6fec7f1add6e46942d070cb779c4fd512455636e0R172.

Would you say I should change also `egammaObjectModificationsInMiniAOD_cff.py` to stay consistent?

Reply:
the meaning of the modifiers and even the set of them changed since July

Reply:
So does that mean I should do the same change you suggested here also in https://github.com/cms-sw/cmssw/pull/30255/files#diff-f273dbbb39fe5fe2475dabb6fec7f1add6e46942d070cb779c4fd512455636e0R172?

Reply:
I'll suppose yes

Reply:
Ah my bad, I see it now! It has been changed already.